.github/workflows/are-we-compiled-yet.yml

```diff
@@ -18,11 +18,11 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
 
-      - uses: pnpm/action-setup@v2
+      - uses: pnpm/action-setup@v4
         name: Install pnpm
         id: pnpm-install
         with:
-          version: 8
+          version: 8.15.8
```

Comment:
It looks like this [field is optional](https://github.com/pnpm/action-setup/blob/23657c8550e9be4f7ee7e9d7fe8adda6d703bb4e/README.md?plain=1#L11) since we have `packageManager` specified in `package.json` 🤔  
```suggestion
```